lib/scala/common-polyglot-core-utils/src/main/java/org/enso/polyglot/common_utils/Core_Math_Utils.java

```diff
@@ -31,8 +37,7 @@ public class Core_Math_Utils {
    * @throws IllegalArgumentException if `n` is outside the allowed range.
    * @throws IllegalArgumentException if `decimalPlaces` is outside the allowed range.
    */
-  public static double roundDouble(double n, int decimalPlaces, boolean useBankers) {
-
+  public static double roundDouble(double n, long decimalPlaces, boolean useBankers) {
```

Comment:
Is this a copy of the code in `RoundNode`? Shall not we seek a way to share the code among both versions?

Reply:
After discussing it, we decided it was sufficient to share the tests; sharing the implementation between builtins and regular polyglot libraries is a bigger task.